tests/components/huawei_lte/test_config_flow.py

```diff
@@ -181,13 +181,13 @@ def login_requests_mock(requests_mock):
     )
     config_entry.add_to_hass(flow.hass)
 
-    result = await flow.async_get_options_flow(config_entry).async_step_init()
+    result = await flow.hass.config_entries.options.async_init(config_entry.entry_id)
```

Comment:
We should remove the flow fixture and replace with core interface from `hass`. This can be done in a follow up PR if preferred.